experimental/dds/tree2/src/core/edit-manager/editManager.ts

```diff
@@ -96,6 +105,8 @@ export class EditManager<
 		};
 		this.trunk = this.trunkBase;
 		this.localBranch = this.trunk;
+
+		// Do not call `getHead` for either of the undo redo managers within this constructor.
```

Comment:
If you want this comment here, it can stay, just bumping this in case you meant to remove it and forgot.

Reply:
I did forget. Thank you :smile: